.gitignore

```diff
@@ -0,0 +1 @@
+.idea/*
```

Comment:
Not sure if we really need that, this project is essentially Dockerfiles and a YAML file.

Reply:
Agree, no need to add the .gitignore file.